kernel/build/config-amd64

```diff
@@ -611,28 +612,34 @@ CONFIG_ACPI_THERMAL=y
 CONFIG_ARCH_HAS_ACPI_TABLE_UPGRADE=y
 CONFIG_ACPI_TABLE_UPGRADE=y
 # CONFIG_ACPI_DEBUG is not set
-# CONFIG_ACPI_PCI_SLOT is not set
+CONFIG_ACPI_PCI_SLOT=y
 CONFIG_ACPI_CONTAINER=y
 CONFIG_ACPI_HOTPLUG_MEMORY=y
 CONFIG_ACPI_HOTPLUG_IOAPIC=y
 # CONFIG_ACPI_SBS is not set
-# CONFIG_ACPI_HED is not set
-# CONFIG_ACPI_BGRT is not set
+CONFIG_ACPI_HED=y
+CONFIG_ACPI_BGRT=y
```

Comment:
Do we really need this? Iirc that's a table supplying UEFI vendor Logo for OS boot screen

Reply:
I can disable that as well, I thought it might be cool eventually to support showing this :)

Reply:
well, yes, ACPI table parsing shouldn't be heavy so okay

Reply:
well i don;t think it hurts to show the vendor logo